spec/features/super_admin/lookup_wifi_user_contact_details_spec.rb

```diff
@@ -45,7 +45,7 @@
   end
 
   context "with a phone number search term" do
-    let(:search_term) { "+44 7891 234567" }
+    let(:search_term) { "+447891234567" }
```

Comment:
Should it still test phone numbers with spaces, to catch that scenario, ie ensure spaces are ignored/removed ?

Reply:
It won't handle spaces in phone numbers but I don't think it matters. This way it _does_ handle names with spaces in it